### ISSUE TITLE
Don't show empty post version diffs (by default?)

### ISSUE THREAD

**Issue by nonamethanks** [danbooru/danbooru#4354]
Example: https://danbooru.donmai.us/post_versions?search%5Bpost_id%5D=3834390

This problem is common because adding and then removing a tag results in an empty version diff. This creates clutter in the post versions pages, so they should not be displayed by default.

**Comment by BrokenEagle** [comment]
There should only be an option to filter these out with some search option. This would be possible with post versions and maybe pool versions, but none of the other versions are really set up for this.

**Comment by nonamethanks** [comment]
Another reason to have this is that now the tag changes report takes a lot more space (https://danbooru.donmai.us/post_versions?search%5Bupdater_id%5D=508240&search%5Bversion%5D=1&type=current) so it's hard to see the ones that actually changed.

**Comment by BrokenEagle** [comment]
> Another reason to have this is that now the tag changes report takes a lot more space

It should actually be the same amount of space. The only difference is what columns are being shown, i.e. with this it shows the additional columns because it is the post versions index.

> so it's hard to see the ones that actually changed.

It was always difficult to see the ones that changed if you didn't have a lot of posts with changed tags.

**Comment by nonamethanks** [comment]
Yeah but before it was cleaner, because you could quickly see the posts with changed tags at a glance, and assess the "quality" of tagging of an user, so to speak. Now it's not immediate anymore.

**Comment by nonamethanks** [comment]
Closing this because I don't think it's a good idea anymore.
